src/engine/MaxConstraint.cpp

```diff
@@ -332,7 +376,7 @@ void MaxConstraint::resetMaxIndex()
 List<PiecewiseLinearConstraint::Fix> MaxConstraint::getPossibleFixes() const
 {
     ASSERT( !satisfied() );
-    ASSERT( _assignment.exists( _f ) && _assignment.size() > 1 );
+    ASSERT( _assignment.exists( _f ) && _assignment.size() > 1 ); // todo Guy A - 2nd assert is still relevant?
```

Comment:
maybe change it to 

 _assignment.exists( _f ) && ( _assignment.size() > 1 || _haveEliminatedVariables )